modules/apps/portal-search/portal-search-rest-test/src/testIntegration/java/com/liferay/portal/search/rest/resource/v1_0/test/SearchResultResourceTest.java

```diff
@@ -231,34 +236,84 @@ private void _assertFacetConfiguration(
 		_assertFacetConfiguration(null, facetName, facetValues, expectedValues);
 	}
 
-	private Page<SearchResult> _postSearchPage(String keywords)
+	private String _getEndpointWithParameters(
+		String entryClassNames, String filter, String nestedFields,
```

Comment:
- Please check the parameter order: 'keywords' after 'filter'.
- A matter of taste, but I'd find the method name like getEndpointURL() a little more descriptive